src/AccordionContainer/AccordionContainer.tsx

```diff
@@ -19,13 +19,15 @@ export type ProviderState = {
 
 export type ProviderProps = {
     allowMultipleExpanded?: boolean;
+    allowZeroExpanded?: boolean;
     children?: React.ReactNode;
     items?: Item[];
     onChange?(args: UUID[]): void;
 };
 
 export type AccordionContainer = {
     allowMultipleExpanded: boolean;
+    allowZeroExpanded?: boolean;
```

Comment:
I don't think that this is an optional (`?`) property of this type. It _is_ optional as a prop though (`ProviderProps`), just not here.